### PR TITLE
fix: Update token component to match GD v1.05

### PR DESCRIPTION
**Pull request by saad-mo**
Closes sap/fundamental#952

Updated token component to match GD v1.05

before:
Screen Shot 2019-04-19 at 9 54 33 AM

after: 
Screen Shot 2019-04-19 at 9 53 22 AM

#### Test
* http://localhost:3030/token
* http://localhost:4000/components/token.html

#### Changelog

**New**

nothing

**Changed**

* updated bg color to `neutral 1`
* updated border color from `transparent` to `neutral 3`
* change close icon from `sys-cancel` to `decline`

**Removed**

nothing

### CODE REVIEW

**Comment by saad-mo** [comment]
@LeoT7508 for your review - https://deploy-preview-1409--fundamental.netlify.com/components/token.html

File: test/templates/token/data.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "Token",
     "css_vars": true,
     "rtl": true,
-    "version": "1.0.0",
+    "version": "1.0.5",
```
Comment (eboyer) [inline test/templates/token/data.json]: This brings up a good point. Are we trying to still track versions on individual components?

Comment (saad-mo) [reply test/templates/token/data.json]: we can discuss... we have to figure out the value of version number on components. i thought we could match it to the GD master sketch file, but according to @LeoT7508 there isn't any versioning that is happening right now.

**Comment by LeoT7508** [comment]
@saad-mo 

2 mine updates. We need to change the color of the idle token text to Text 1 and add in the states, hover, selected, read only, display only.

**Comment by saad-mo** [comment]
closing to solve CLA Issue

**Pull request closed without merging**